uber/models.py

```diff
@@ -700,13 +700,15 @@ def delete_from_group(self, attendee, group):
             self.delete(attendee)
             group.attendees.remove(attendee)
 
-        def assign_badges(self, group, new_badge_count, new_badge_type=None, **extra_create_args):
-            new_badge_type = new_badge_type or c.ATTENDEE_BADGE
+        def assign_badges(self, group, new_badge_count, new_badge_type=c.ATTENDEE_BADGE, new_ribbon_type=c.NO_RIBBON, **extra_create_args):
             diff = int(new_badge_count) - group.badges
             sorted_unassigned = sorted(group.floating, key=lambda a: a.registered, reverse=True)
+
+            ribbon_to_use = group.new_ribbon if group.new_ribbon is not c.NO_RIBBON else new_ribbon_type
```

Comment:
The way this is written, there's literally no way to override the default value provided by ``group.new_ribbon``.  If someone deliberately selects a ribbon type in the dropdown, they'd presumably expect the ribbon type they select to be used.  I realize you're hiding it for new dealers, but admins often add badges to existing groups.  Most of the time they probably won't want to add badges to the group with a different ribbon type than the default, but it seems like it should still be possible.

I'd probably make ``new_ribbon_type=None`` and then rewrite this line to read something like

```python
ribbon_to_use = new_ribbon_type or group.new_ribbon
```